app/controllers/v6/assessments_controller.rb

```diff
@@ -141,6 +145,28 @@ def parse_employment_details(employments)
       end
     end
 
+    def parse_employment_income(employments, submission_date)
+      employments.map do |employment|
```

Comment:
as we're using models, we should freeze them to make sure that they are immutable through their lifetime